sbndqm/Decode/PMT/SBND/DaqDecoderSBNDPMT_module.cc

```diff
@@ -221,7 +221,7 @@ void daq::DaqDecoderSBNDPMT::processFragment( const artdaq::Fragment &artdaqFrag
   const uint16_t* data_begin = reinterpret_cast<const uint16_t*>(artdaqFragment.dataBeginBytes() + sizeof(sbndaq::CAENV1730EventHeader));
 
   std::vector<uint16_t> wvfm(nSamplesPerChannel);
-  float temperature = 0;
+  std::vector<float> temperature;
```

Comment:
It looks like it became many:
```suggestion
  std::vector<float> temperatures;
```